python/tests/pytorch/test_torchvision.py

```diff
@@ -30,7 +30,9 @@ def test_vision_dataset(spark: SparkSession, tmp_path: Path):
     df = _create_dataframe(tmp_path, spark)
     df.show()
     dataset_dir = str(tmp_path / "data")
-    df.write.mode("overwrite").format("rikai").save(dataset_dir)
+    df.write.mode("overwrite").format("rikai").mode("overwrite").save(
```

Comment:
Why does this have even more `overrite` ;p

Reply:
Fixed, the vanilla regex doesn't support semantic analysis, and I rely on it too much 😉